src/mmrelay/meshtastic/health.py

```
@@ -0,0 +1,746 @@
+                    )
+            else:
+                try:
+                    # NOTE: Use the metadata admin request for keepalive/liveness.
+                    # `getMyNodeInfo()` is local cached state in Meshtastic Python,
+                    # so it can succeed even when the transport is unhealthy.
+                    await facade.asyncio.wait_for(
+                        facade.asyncio.wrap_future(probe_future),
+                        timeout=probe_timeout,
+                    )
+
+                except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)